src/GitLabApiClient/Models/Files/Requests/FileQueryOptions.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GitLabApiClient.Models.Files.Requests
+{
+    public sealed class FileQueryOptions
+    {
+        public string Branch { get; set; }
```

Comment:
Branch is not accurate, it can be a reference which points to Branch, Tag or commit sha
```suggestion
        public string Reference { get; set; }
```